certified-connectors/Impower/apiDefinition.swagger.json

```diff
@@ -610,16 +612,18 @@
           {
             "name": "type",
             "in": "query",
-            "description": "Type of the contract. Options: OWNER | TENANT | TENANT_VACANCY | PROPERTY_OWNER",
-            "x-ms-summary": "Type of the contract. Options: OWNER | TENANT | TENANT_VACANCY | PROPERTY_OWNER",
+            "description": "Type of the contract. Options: OWNER | TENANT | PROPERTY_OWNER",
+            "x-ms-summary": "Type of the contract. Options: OWNER | TENANT | PROPERTY_OWNER",
             "required": false,
-            "type": "string",
```

Comment:
The type has been changed from string to array, can you please confirm that no existing flows or users will be affected by this change?

Reply:
this API is currently not deployed to production.
doesn't have any customer impact.